helmfile/values/networkpolicies/workload/alertmanager.yaml.gotmpl

```diff
@@ -37,7 +37,7 @@ policies:
             - namespaceSelectorLabels:
                 kubernetes.io/metadata.name: falco
               podSelectorLabels:
-                app.kubernetes.io/name: falco-sidekick
+                app.kubernetes.io/name: falcosidekick
```

Comment:
Do we have some netpol in the falco namespace that needs to be update as well or is that already done?

Reply:
seems to be correct there [sc](https://github.com/elastisys/compliantkubernetes-apps/blob/QA-0.31/helmfile/charts/networkpolicy/service-cluster/templates/falco/allow-falco-sidekick-sc.yaml) and [wc](https://github.com/elastisys/compliantkubernetes-apps/blob/QA-0.31/helmfile/charts/networkpolicy/workload-cluster/templates/falco/allow-falco-sidekick-wc.yaml#L14)